docs/Installation-iOS.md

```diff
@@ -4,16 +4,17 @@ This guide will help you install Theos on your jailbroken iOS device.
 |----------|--------------------|-------------------|
 | **iOS** | 5.0 | iOS |
 
-All the commands shown in the following instructions are meant to be run as the "user" user, _not_ **root**. Similarly, Theos is also meant to be run as a normal user, _not_ **root**.
+All the commands shown in the following instructions are meant to be run as the "mobile" user, _not_ **root**. Similarly, Theos is also meant to be run as a normal user, _not_ **root**.
 
 1. Install the following prerequisites:
 
-	* [Sam Bingner’s repository](http://repo.bingner.com/)
-	* Theos Dependencies (package on BigBoss repo, relies on the previous repository being installed first)
+	* [Procursus](https://apt.procurs.us/) or [Sam Bingner’s repository](http://repo.bingner.com/)
+	* Theos Dependencies (package on either the Procursus or BigBoss repo, relies on the previous repository being installed first)
 
 1. Set up the `THEOS` environment variable:
 
 		echo "export THEOS=~/theos" >> ~/.profile
+		echo "export THEOS=~/theos" >> ~/.zprofile
```

Comment:
If we're going to explicitly support zsh, we should do so on macOS as well since it's the new default since Catalina. We should explain that you are not supposed to modify both files if you use only one shell.

Reply:
I don't know if this is a "supposed to" modify moment. It doesn't really affect anything if you modify both.

Reply:
`if [ "$(echo "$SHELL" | grep -qi zsh && echo 1)" = "1" ]; then PROFILE=.zprofile; else PROFILE=.profile; fi; echo "export THEOS=~/theos" >> ~/$PROFILE`

This is a bit more complex, but covers the common bases in one copy/paste.

Reply:
For simplicity why don't we split this step into two headings: zsh and bash. And maybe put $THEOS in .zshenv instead of .zprofile?

Reply:
@kabiroberai 👍

Reply:
I think this is what I meant, sorry for the confusion